crypto/src/asset.rs

```diff
@@ -36,6 +38,7 @@ impl std::fmt::Debug for Id {
 
 // XXX define a DenomTrace structure ?
 
+// xx rename this derive?
```

Comment:
long-run I think we should probably eliminate it and have `From` for a new `Denom` type.